app/models/user.rb

```diff
@@ -1,12 +1,10 @@
 # frozen_string_literal: true
 
 class User < ApplicationRecord
-  ADMIN = 'admin'
-  CUSTOMER = 'customer'
-
-  ROLES = [ADMIN, CUSTOMER].freeze
-
-  enum role: ROLES
+  enum role: ROLES = {
```

Comment:
you don't really need to have that constant due to the possibility to access the rules list via `User.roles`